src/Symmetric/Mcrypt.php

```diff
@@ -145,8 +165,18 @@ public function __construct($options = [])
                         break;
                 }
             }
+            $this->options = $options;
         }
-        $this->setDefaultOptions($options);
+    }
+
+    /**
+     * Get the Options
+     *
+     * @return array
+     */
+    public function getOptions()
```

Comment:
Is the getter necessary? i.e., is there a use case for retrieving the current set of options, or does it primarily benefit testing?